feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -1,42 +1,14 @@
 <ng-container *ngIf="configuration$ | async as configuration">
```

Comment:
Is it really intended that sub groups and parent groups are styled that differently?
![Screen Shot 2021-05-27 at 12 14 18](https://user-images.githubusercontent.com/33626130/119809508-7be00800-bee5-11eb-9f1c-91736972beae.png)

Reply:
Yes I will clarify it with Anthony today. But in the visuals they are styled differently.
In the future it is planned to collapse it on top level group. 
https://user-images.githubusercontent.com/51401884/117640208-10780580-b185-11eb-8492-1e58bccbd47e.png